### PR TITLE
make and use new component GroupElement

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
- Make component GroupElements that takes react components as children and renders the elements besides eachother.
- I think the API of GroupElements is better than DualElements and NavigationButtons component as the component is more flexible.
- Delete DualElementsContainer and NavigationButtons
- Move all navigation buttons to the left
- Added "finished"-button to single rights ReceiptPage

## Related Issue(s)
- #542 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/components/GroupElements/GroupElements.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ *  <Button>Previous</Button>
+ *  <Button>Next</Button>
+ * </GroupElements>
+ */
+
+export const GroupElements = ({ children }: GroupElementsProps) => {
+  return (
+    <ul className={classes.list}>
+      {children.map((child: React.ReactNode, index: number) => (
+        <li
```
Comment (allinox) [inline src/components/GroupElements/GroupElements.tsx:26]: Why display them as list items?

File: src/features/apiDelegation/offered/ConfirmationPage/ConfirmationPage.tsx [hunk 2]
```diff
@@ -30,6 +30,7 @@ export const ConfirmationPage = () => {
   }, [loading]);
 
   const handleConfirm = () => {
+    console.log('Denne gjøres');
```
Comment (allinox) [inline src/features/apiDelegation/offered/ConfirmationPage/ConfirmationPage.tsx:33]: Did you mean to push this? 😅

File: src/localizations/en.json [hunk 3]
```diff
@@ -32,7 +32,8 @@
     "read": "Read",
     "write": "Write",
     "sign": "Sign",
-    "access": "Access to whole service"
+    "access": "Access to whole service",
```
Comment (allinox) [inline src/localizations/en.json:35]:
```suggestion
    "access": "Access to the whole service",
```

**Review by allinox — APPROVED** [review]
Good job! 😊
I like the new groupElements component - I agree that it is a much more elegant solution than the previous two components

File: src/components/GroupElements/GroupElements.tsx (hunk 1 above)
Comment (Albertlarsen) [reply src/components/GroupElements/GroupElements.tsx:26]: Design system does that for chip group and I think it is a common way to show semantically that two components are within the same group.

File: src/features/apiDelegation/offered/ConfirmationPage/ConfirmationPage.tsx (hunk 2 above)
Comment (Albertlarsen) [reply src/features/apiDelegation/offered/ConfirmationPage/ConfirmationPage.tsx:33]: Hehe no

File: src/components/GroupElements/GroupElements.tsx (hunk 1 above)
Comment (allinox) [reply src/components/GroupElements/GroupElements.tsx:26]: Ok, that makes sense 👍

**Pull request merged**